walkthroughs/week-4-tdd/intro/src/main/java/com/google/sps/Greeter.java

```diff
@@ -22,6 +22,11 @@
    * Returns a greeting for the given name.
    */
   public String greet(String name) {
-    return "Hello " + name;
+    String trimName = name.trim();
+    String noSymbols = trimName.replace("@", "");
+    noSymbols = noSymbols.replace("#", "");
+    noSymbols = noSymbols.replace("$", "");
+    noSymbols = noSymbols.replace("%", "");
+    return "Hello " + noSymbols;
```

Comment:
Might consider putting a comment either here or in the function's docstring, just explaining that you're doing some preprocessing on the name. Also, if i put in the name "@@@     Jef    %", what would the result be?